docs/getting-started.md

```diff
@@ -22,26 +22,64 @@ yarn add offix-client
 
 ```javascript
 import { OfflineClient } from 'offix-client';
-```
 
-## Configuration
+const config = {
+  httpUrl: 'http://localhost:4000/graphql'
+};
 
-To provide custom configuration to the client, the following options are available. If you wish, these are also available by using the `OffixClientConfig` interface from the SDK.
+// offlineClient is a parent around the apollo client
+const offlineClient = new OfflineClient(config);
 
-```javascript
-let config = {
-  httpUrl: 'http://localhost:4000/graphql',
-  wsUrl: 'ws://localhost:4000/graphql'
+// client is the initialized apollo client
+const client = await client.init();
```

Comment:
In future we will need to probably give more react/angular examples with progressbar